### PR TITLE
Feat/private playlists in sidebar

### PR DESCRIPTION
**Pull request by belljohan3**

### CODE REVIEW

File: src/components/PlaylistOverview.vue [hunk 1]
```diff
@@ -1,48 +1,41 @@
...
-import { useAuth0 } from "@auth0/auth0-vue";
+import {
+  PlaylistApi,
+  PlaylistModel,
+  Configuration,
+} from "@bcc-code/bmm-sdk-fetch";
 
 const playlists: Ref<PlaylistModel[]> = ref([]);
 
-const { getAccessTokenSilently } = useAuth0() ?? {
```
Comment (kkuepper) [inline src/components/PlaylistOverview.vue:11]: I think a merge went bad since you're undoing a lot of changes from other people

**Review by kkuepper — CHANGES_REQUESTED** [review]

File: src/components/PlaylistOverview.vue (hunk 1 above)
Comment (belljohan3) [reply src/components/PlaylistOverview.vue]: The commit has been fixed ! Please can have a look again ? Thanks

**Review by SimonSimCity — COMMENTED** [review]
Before merging, I'd like having the comments and the `console.log()` removed, if not needed. For the token, I'd either wait for #47 or fix it there if this is merged earlier.

File: src/components/privatePlaylist.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    // eslint-disable-next-line no-console
+    console.log(list);
+  })
+  .catch(() => {});
+</script>
+
+<template>
+  <div>
+    <h3>Playlists</h3>
+    <ul v-for="collection in collections" :key="collection.id || 0">
```
[the remark this answers is not in the document]
Comment (belljohan3) [reply src/components/privatePlaylist.vue:33]: Okay !

File: src/components/privatePlaylist.vue [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      <li>
+        <a href="">{{ collection.name }}</a>
```
Comment (SimonSimCity) [inline src/components/privatePlaylist.vue:35]: Would be nice to here have links to the route "PrivatePlaylist". The commit fbd733726515a29ba3a6585fccf4a6da5566693e should give you a good start :)

**Comment by kkuepper** [comment]
What's the status on this? Would be nice to get this merged.

File: src/components/privatePlaylist.vue [hunk 4]
```diff
@@ -1,38 +1,37 @@
...
 
 <template>
   <div>
     <h3>Playlists</h3>
-    <ul>
-      <li v-for="collection in collections" :key="collection.id || 0">
-        <a href="">{{ collection.name }}</a>
-      </li>
-    </ul>
+    <RouterLink
```
Comment (SimonSimCity) [inline src/components/privatePlaylist.vue:18]: Could you move the router-link to be inside the list-item (the `li` tag)? As of HTML, you're now creating a new list for every item, whereby I'd argue that all these should be an item on the same list 😏

File: src/views/Playlist/CuratedPlaylistDetails.vue [hunk 5]
```diff
@@ -34,8 +34,8 @@ getTracks(playlistId)
     <h3>{{ playlist.title }}</h3>
     <br />
     <ol class="list-decimal list-inside">
-      <li v-for="track in tracks" :key="track.id || 0">
-        {{ track.meta?.artist }} - <b>{{ track.meta?.title }}</b>
+      <li v-for="track in tracks" :key="track.id || 0" class="flex">
+        {{ track.meta?.title }} - <b>{{ track.meta?.artist }}</b>
```
Comment (SimonSimCity) [inline src/views/Playlist/CuratedPlaylistDetails.vue:38]: These changes do not seem to either be part of a configuration or routes, are they ..?

File: src/views/Playlist/PrivatePlaylist.vue [hunk 6]
```diff
@@ -17,10 +17,26 @@ get(Number(props.id))
...
-  <h1>{{ trackCollection?.name }}</h1>
+  <h2>{{ trackCollection?.name }}</h2>
   <ol class="list-decimal list-inside">
     <li v-for="track in trackCollection.tracks" :key="`track-${track.id}`">
       {{ track.title }}
     </li>
   </ol>
 </template>
+
+<style scoped>
```
Comment (SimonSimCity) [inline src/views/Playlist/PrivatePlaylist.vue:28]: Could you revert these changes? I don't know if they're in line with the styleguide - if it is, it shouldn't be done this way (scoped, custom styles).

File: src/components/privatePlaylist.vue (hunk 4 above)
Comment (belljohan3) [reply src/components/privatePlaylist.vue:18]: Ok. How can I manage with the properties please ?

File: src/views/Playlist/PrivatePlaylist.vue (hunk 6 above)
Comment (SimonSimCity) [reply src/views/Playlist/PrivatePlaylist.vue:28]: Maybe, I don't know much about tailwind, but I wouldn't see it as part of the problem you're creating this PR for.

**Comment by belljohan3** [comment]
@SimonSimCity and @kkuepper i think i am done with this task ! Could you please have a look at this pull request ?

File: src/views/Playlist/PrivatePlaylist.vue (hunk 6 above)
Comment (SimonSimCity) [reply src/views/Playlist/PrivatePlaylist.vue:31]: Nothing this time - just saying that this is not a good way of doing it. We'll have a round of design-changes to the project in a later state, where we'll just concentrate on getting the design straight and define a way of how to do it. I'm not the right person to give a review here...

File: src/components/privatePlaylist.vue [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+  .catch(() => {});
+</script>
+
+<template>
+  <div>
+    <h3>Playlists</h3>
+    <ul>
+      <li>
+        <RouterLink
+          v-for="collection in collections"
```
Comment (SimonSimCity) [inline src/components/privatePlaylist.vue:24]: With these changes, we have one unordered list (which is good) but this one list has also just only one list-item, which contains the many playlists the user might have. Each playlist should (as a list has multiple list-items) be on a separate list-item.

**Review by SimonSimCity — CHANGES_REQUESTED** [review]

**Review by SimonSimCity — APPROVED** [review]

**Review by kkuepper — APPROVED** [review]

**Comment by kkuepper** [comment]
Awesome! @belljohan3 
Thanks for you contribution!

**Comment by belljohan3** [comment]
Thanks so much @kkuepper ! I would like to contribute more and more.

**Pull request merged**